test/examples/star-cors-example.test.ts

```diff
@@ -29,4 +29,13 @@ describe('Star CORS examples', () => {
 
     response.headers['access-control-allow-origin'].should.eql('http://localhost:3000');
   });
+
+  it('/api triggers cors policy', async () => {
```

Comment:
Let's add one more test with a complext domain that the cors succeeds.

Reply:
Done